swan-lake/development-tutorials/java-interoperability/ballerina-ffi.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: ballerina-java-interoperability-left-nav-pages-swanlake
 title: Ballerina FFI
-description: The reference guide on the list of language features that enable Ballerina developers to call foreign code written in other programming languages.
+description: This is a reference guide on the list of language features that enable Ballerina developers to call foreign code written in other programming languages.
 keywords: ballerina, programming language, ffi, foreign function invocation
 permalink: /learn/java-interoperability/ballerina-ffi/
 active: ballerina-ffi
-intro: The reference guide on the list of language features that enable Ballerina developers to call foreign code written in other programming languages.
+intro: This is a reference guide on the list of language features that enable Ballerina developers to call foreign code written in other programming languages.
 ---
 
-Let's look at the list of language features that enable Ballerina developers to call foreign code written in other programming languages. E.g., while the jBallerina compiler allows you to call any `Java` code, the nBallerina compiler allows you to call any `C` Code.
+For example, while the jBallerina compiler allows you to call any `Java` code, the nBallerina compiler allows you to call any `C` Code.
```

Comment:
Why `Let's look at the list of language features that enable Ballerina developers to call foreign code written in other programming languages` part is removed? IMO, it is incomplete to start with `For example` without that sentence.  @praneesha

Reply:
The `the list of language features that enable Ballerina developers to call foreign code written in other programming languages.` part appears incorrectly in 2 places on the page because it is repeated in the description and the body of the page. It is correct to have it for the `description` and `intro` and that's why it was removed from the page body.

It is fine to start the body with `For example` as the `description will also be shown before it when rendered.